plugins/out_es/es.c

```diff
@@ -890,11 +890,12 @@ static void cb_es_flush(struct flb_event_chunk *event_chunk,
         flb_http_basic_auth(c, ctx->cloud_user, ctx->cloud_passwd);
     }
     else if (ctx->http_api_key) {
-        header_line = flb_sds_printf(NULL, "ApiKey %s", ctx->http_api_key);
+        header_line = flb_sds_create_size(48);
```

Comment:
Do we know the API key will consistently be 48 bytes? Maybe should use the length of the value in the field for this.

```suggestion
        header_line = flb_sds_create_size(strlen(ctx->http_api_key)+48);
```

Reply:
I was expecting ES API key are always 40 bytes (+8 for `ApiKey `) but this does seem safer, just in case